examples/drug_gripnet/README.md

```diff
@@ -1,13 +1,39 @@
-# Link Prediction: GripNet Framework on PoSE Sample Dataset
+# Polypharmacy Side Effect Prediction using GripNet (Link Prediction)
 
 ## 1. Description
-This example is constructed by [GripNet Framework](https://arxiv.org/abs/2010.15914) on Polypharmacy Side Effect (PoSE) prediction task. The PoSE sample dataset is from BioSNAP-Decagon dataset and the target is to predict relations between drug nodes on the whole gene-drug heterogeneous graph.
+
+Polypharmacy is the concurrent use of multiple medications by patients. Its side effects, polypharmacy side effects, are caused by drug combinations rather than by any single drug. In this example, we train a GripNet model [1] for predicting polypharmacy side effects.
+
+**Reference**:
+
+[1] Xu, H., Sang, S., Bai, P., Yang, L., & Lu, H. (2022). [GripNet: Graph Information Propagation on Supergraph for Heterogeneous Graphs](https://doi.org/10.1016/j.patcog.2022.108973). Pattern Recognition.
+
+## 2. GripNet
+
+**Gr**aph **i**nformation **p**ropagation **Net**work (GripNet) is an effective and efficient framework for learning node representations on knowledge graphs (Subfigure A) for link prediction and node classification. It introduces a novel supergraph structure (Subfigure B) and constructs a graph neural network that passes messages via a task-specific propagation path based on the supergraph (Subfigure C).
+
+![GripNet Architecture](https://ars.els-cdn.com/content/image/1-s2.0-S0031320322004538-gr2_lrg.jpg)
 
 The original implementation of GripNet is [here](https://github.com/NYXFLOWER/GripNet.git).
 
-## 2. Usage
-* Dataset download: [PoSE](https://github.com/pykale/data/tree/main/graphs)
+## 3. Dataset
+
+We train GripNet on three datasets constructed by [2].
+The datasets are downloaded from the [Bio-SNAP](http://snap.stanford.edu/biodata/) dataset collection. The integration of these datasets can be regarded as a knowledge graph, which contains the relations among proteins (P) and drugs (D) labeled with the P-P association, the P-D association and 1,317 side effects. The data statistics are shown below.
+
+| Dataset           | Nodes            | Edges   | #Unique Edge Labels |
+| ----------------- | ---------------- | ------- | ------------------- |
+| PP-Decagon        | 19,081(P)        | 715,612 | 1                   |
+| ChG-TargetDecagon | 3,648(P), 284(D) | 18,690  | 1                   |
+| ChChSe-Decagon    | 645(D)           | 63,473  | 1,317               |
+
+**Reference**:
```

Comment:
Maybe group the two references and put at the bottom? You can still add hyperlink to paper above in description.